orc8r/cloud/deploy/terraform/orc8r-helm-aws/helm.tf

```diff
@@ -25,7 +25,7 @@ resource "kubernetes_service_account" "tiller" {
 
 # helm tiller cluster role
 resource "kubernetes_cluster_role_binding" "tiller" {
-  count = var.existing_tiller_service_account_name == null ? 1 : 0
+  count = var.install_tiller == true ? 1 : 0
```

Comment:
Same as before w.r.t tiller deprecation in the future. I understand why we don't need a new tiller object. but not quite clear on the variable name changing here.

Reply:
Previously we were using both existing_tiller_service_account_name & orc8r_is_staging_deployment to disable tiller.

Hunter proposed to use just the one above and configure it all in main.tf